src/Writer.cpp

```diff
@@ -25,12 +25,12 @@ Writer::Writer() {
 std::map<std::string, ValueSerializer> Writer::loadDefaultValueSerializers() {
   // clang-format off
   std::map<std::string, ValueSerializer> serializer_functions = {
-    {Pose2Tag, [](gtsam::Key key, gtsam::Values& vals) { return serialize<gtsam::Pose2>(vals.at<gtsam::Pose2>(key)); }},
-    {Pose3Tag, [](gtsam::Key key, gtsam::Values& vals) { return serialize<gtsam::Pose3>(vals.at<gtsam::Pose3>(key)); }},
-    {Point2Tag, [](gtsam::Key key, gtsam::Values& vals) { return serialize<gtsam::Point2>(vals.at<gtsam::Point2>(key)); }},
-    {Point3Tag, [](gtsam::Key key, gtsam::Values& vals) { return serialize<gtsam::Point3>(vals.at<gtsam::Point3>(key)); }},
-    {VectorTag, [](gtsam::Key key, gtsam::Values& vals) { return serialize<gtsam::Vector>(vals.at<gtsam::Vector>(key)); }},
-    {ScalarTag, [](gtsam::Key key, gtsam::Values& vals) { return serialize<double>(vals.at<double>(key)); }},
+    {Pose2Tag,  [](gtsam::Key key, gtsam::Values& vals) { return io_values::serialize<gtsam::Pose2>(vals.at<gtsam::Pose2>(key)); }},
```

Comment:
namespace specification is not needed since there is a `using namespace io_values` in this file. Given the namespace is included we should remove these tags.